packages/core/getBusyTimes.ts

```diff
@@ -69,9 +73,14 @@ export async function getBusyTimes(params: {
    */
   performance.mark("prismaBookingGetStart");
 
+  const startTimeDate =
+    rescheduleUid && duration ? dayjs(startTime).subtract(duration, "minute").toDate() : new Date(startTime);
```

Comment:
On the time of booking a slot (handleNewBooking) `startTime` is the start time of the new booking, which means that the old booking would not be included in `bookings`. If we add the duration of the original booking it should always be included when querying `bookings`